frontend/src/model/beta/registry/ModelImageDisplay.tsx

```diff
@@ -0,0 +1,58 @@
+import { Box, Stack, Typography } from '@mui/material'
+import { useTheme } from '@mui/material/styles'
+import { useGetUiConfig } from 'actions/uiConfig'
+import { useMemo } from 'react'
+import Loading from 'src/common/Loading'
+import MessageAlert from 'src/MessageAlert'
+import CodeLine from 'src/model/beta/registry/CodeLine'
+
+import { ModelImage } from '../../../../types/interfaces'
+
+type ModelImageDisplayProps = {
+  modelImage: ModelImage
+}
+
+export default function ModelImageDisplay({ modelImage }: ModelImageDisplayProps) {
+  const theme = useTheme()
+
+  const { uiConfig, isUiConfigLoading, isUiConfigError } = useGetUiConfig()
+
+  const modelImageTags = useMemo(
+    () =>
+      uiConfig &&
+      modelImage.tags.map((imageTag) => (
+        <CodeLine
+          key={`${modelImage.name}-${imageTag}`}
```

Comment:
This will also need the repository to make it unique
`key={`${modelImage.repository}-${modelImage.name}-${imageTag}`}`